projects/storefrontlib/src/cms-structure/guards/cms-page.guard.ts

```diff
@@ -75,15 +74,17 @@ export class CmsPageGuard implements CanActivate {
         }
       }),
       map(([canActivate, componentTypes]) => {
+        const pageLabel = pageData.label;
```

Comment:
what about 
`const pageLabel = pageData.label || pageContext.id;`
for backward compatibility?

Reply:
Moreover, it's necessary to use original ID for non-content pages (PDP & PLP), because they don't send `label` in the response.